app/javascript/src/stylesheets/shared/layout.scss

```diff
@@ -1,5 +1,5 @@
 body {
-  background-color: #F4F7FA;
```

Comment:
this is an ok change between it doesn't change the actual color :)

Reply:
Ok.  My linter made this change.  I will disable the lint on save moving forward.